docs/providers/aws/guide/deploying.md

```diff
@@ -78,7 +78,7 @@ This deployment method does not touch your AWS CloudFormation Stack.  Instead, i
 serverless deploy function --function myFunction
 ```
 
-**Note:** You can always enforce a deployment using the `--force` option.
+**Note:** You can always enforce a deployment using the `--force` option or `--update-config` to change function configuration.
```

Comment:
I'd add a separate `**Note:**` section below that since the `--force` and `--update-config` flags provide two distinct functionalities.

Reply:
Sure